tests/integration_tests/allennlp_tests/test_allennlp.py

```diff
@@ -173,6 +173,25 @@ def test_allennlp_executor_with_include_package_arr() -> None:
         assert isinstance(result, float)
 
 
+def test_allennlp_executor_with_options() -> None:
+    study = optuna.create_study(direction="maximize")
+    trial = optuna.trial.Trial(study, study._storage.create_new_trial(study._study_id))
+    trial.suggest_uniform("DROPOUT", 0.0, 0.5)
+
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        executor = optuna.integration.AllenNLPExecutor(
+            trial,
+            "tests/integration_tests/allennlp_tests/example_with_include_package.jsonnet",
+            tmp_dir,
+            force=True,
+            file_friendly_logging=True,
+            include_package=["tests.integration_tests.allennlp_tests.tiny_single_id"],
+        )
+
+        assert executor._force
+        assert executor._file_friendly_logging
```

Comment:
I guess the test may be more solid if we check if these values are used in `executor.run`. How about using the `unittest.mock` as follows?

```python
        from unittest import mock
        with open(os.path.join(executor._serialization_dir, "metrics.json"), "w") as fout:
            json.dump({executor._metrics: 1.0}, fout)

        with mock.patch("allennlp.commands.train.train_model", return_value=None) as mock_obj:
            executor.run()
            assert mock_obj.call_args[1]["force"]
            assert mock_obj.call_args[1]["file_friendly_logging"]
```